components/Snackbar/src/ColorThemer/MDCSnackbarColorThemer.h

```diff
@@ -16,22 +16,19 @@
 
 #import <UIKit/UIKit.h>
 
+#import "MaterialColorScheme.h"
 #import "MaterialSnackbar.h"
-#import "MaterialThemes.h"
 
 /**
- Used to apply a color scheme to theme to MDCSnackbarMessageView.
+ Used to apply a color scheme to theme to MDCSnackbarManager.
  */
 @interface MDCSnackbarColorThemer : NSObject
 
 /**
- Applies a color scheme to theme to a MDCSnackbarMessageView.
- 
- @param colorScheme The color scheme to apply to MDCSnackbarMessageView.
- @param snackbarMessageView A MDCSnackbarMessageView instance to apply a color scheme. Use a
-     UIAppearance proxy to apply a color scheme to all instances of MDCSnackbarMessageView.
+ Applies a color scheme to theme to a MDCSnackbarManager.
+
+ @param colorScheme The color scheme to apply to MDCSnackbarManager.
  */
-+ (void)applyColorScheme:(nonnull id<MDCColorScheme>)colorScheme
```

Comment:
are you sure we want to deprecate it? It was a newly added stub that was added in the last release and has no content. I feel like it would just be more redundant steps on our end.

Reply:
Deprecation as a practice helps external clients migrate to new versions of MDC. I agree that this API is unlikely to have been used, but it's safer for us in general if we always follow the deprecation policy for any public APIs.